src/Blue/Tools/Api/Client.php

```diff
@@ -177,7 +177,19 @@ private function resolve(ResponseInterface $response)
         // An HTTP status of 202 indicates that this request was deferred
         if ($response->getStatusCode() == 202) {
 
-            $key = $response->getBody()->getContents();
+            $content = $response->getBody()->getContents();
+            $res = json_decode($content);
+            if (json_last_error() != JSON_ERROR_NONE) {
+                $key = $content;
+            }
+            else if(isset($res->mailing_triggered_id)) {
+                // There is no need to check the deferred result for triggered mailing.
+                // Once the initial API is successful, we assume that the triggered mailing will be sent properly.
```

Comment:
this comment is too specific to the torchlight use case.  Lets reword it:

// the send_triggered_email API uses 202 to indicate something other than a traditional deferred result